src/test/java/com/meltmedia/jgroups/aws/WithEc2.java

```diff
@@ -0,0 +1,26 @@
+package com.meltmedia.jgroups.aws;
+
+import com.amazonaws.services.ec2.AmazonEC2;
+import com.amazonaws.services.ec2.model.*;
+
+import java.util.Arrays;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public interface WithEc2 {
+  default AmazonEC2 ec2Mock(Tag... tags) {
```

Comment:
This seems like a strange use of default methods.  This should probably be a class called `Mocks` with a `public static AmazonEC2 ec2Mock(Tag... tags)` method and then statically import it into the test.